extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/common/DruidPeonClientIntegrationTest.java

```diff
@@ -54,6 +54,7 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 // must have a kind / minikube cluster installed and the image pushed to your repository
+@Disabled
```

Comment:
We are using K3S now. do you still same issues?

Reply:
k3s?  This was for anyone who had an existing k8s cluster, they could just run this test and point it there.  The k8s integration tests are definitely not flexible enough for me to test mm-less without rewriting a lot of it.

Reply:
how was this running before?

Reply:
I had a kind cluster running locally, it was in the original patch.  For anyone that wants to test with a local k8s cluster I left this test in the codebase.  If you want me to remove it, I'll be happy to get rid of it.  I find it useful for when I make changes, since I can't integration test this with the way current druid-it tests are setup.